runelite-client/src/main/java/net/runelite/client/plugins/microbot/magetrainingarena/MageTrainingArenaConfig.java

```diff
@@ -43,6 +43,18 @@ public interface MageTrainingArenaConfig extends Config {
     )
     String graveyardSection = "graveyard";
 
+    @ConfigItem(
```

Comment:
There's no need to add a config. The mage training arena checks if a player can cast bones to peaches.

        if (btp == null)
            btp = Rs2Magic.canCast(MagicAction.BONES_TO_PEACHES);